lib/ae_page_objects/elements/collection.rb

```diff
@@ -66,10 +73,9 @@ def item_class_at(index)
 
     def item_xpath
       @item_xpath ||= begin
-        evaled_locator = eval_locator(@item_locator)
-        
-        query_args = evaled_locator + [{:exact => true}]
-        query = Capybara::Query.new(*query_args)
+        evaled_locator = eval_locator(@item_locator).clone
+        evaled_locator.pop if evaled_locator.last.is_a?(::Hash)
+        query = Capybara::Query.new(*evaled_locator, options)
```

Comment:
It looks like this change loses the `:exact => true` default. That's a breaking change. I also think it's an undesirable change because it's a good default.